CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Reactist follows [semantic versioning](https://semver.org/) and doesn't introduce breaking changes (API-wise) in minor or patch releases. However, the appearance of a component might change in a minor or patch release so keep an eye on redesigns and make sure your app still looks and feels like you expect it.
 
+# Next-Next
```

Comment:
This may be documented somewhere and I've missed it, but why the need for a Next-Next? Why not add below?